src/PKSim.Core/CoreConstants.cs

```diff
@@ -10,16 +10,13 @@ public static class CoreConstants
       public const int LAYOUT_VERSION = 38;
       public const int DEFAULT_NUMBER_OF_BINS = 20;
       public const int DEFAULT_NUMBER_OF_INDIVIDUALS_PER_BIN = 100;
-      public const int MAX_NUMBER_OF_POINT_PER_INTERVAL = 1000;
```

Comment:
All those parameters are not used anymore. I installed the latest resharper yesterday and I am seeing this info automatically. Neat